setup.cfg

```diff
@@ -162,6 +162,7 @@ isolated_apps =
     openedx.core.djangoapps.agreements
     openedx.core.djangoapps.bookmarks
     openedx.core.djangoapps.content_libraries
+    openedx.core.djangoapps.content_staging
```

Comment:
I am now using [the new `api.py` import linter](https://github.com/openedx/edx-platform/pull/31903) for this new `content_staging` django app; this work was part of my motivation for building that.